File: EPATEC/Models/EmpleadoSucursal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EPATEC.Models
{
    public class EmpleadoSucursal
    {
        public long Id_EmpleadoSucursal { get; set; }
        public long Id_Empleado { get; set; }
        public long Id_Sucursal { get; set; }
    }
}

File: Mobile_App/java/models/EmpleadoSucursal.java
package models;

public class EmpleadoSucursal {
    public long Id_EmpleadoSucursal;
    public long id_empleado;
    public long id_sucursal;

    public EmpleadoSucursal(){}
    public EmpleadoSucursal(long idEmp, long idSuc){
        this.id_empleado = idEmp;
        this.id_sucursal = idSuc;
    }
}
